app/controllers/hq/sponsors_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Hq::SponsorsController < HqController
   def index
-    @sponsors= Sponsor.all
+    @sponsors = Sponsor.paginate(:page => params[:page])
```

Comment:
I think they would, unless you set up a default scope on your model.  Then `Class.paginate` might be a different query.  I would prefer to see us use one or the other, in other words, changing Partner to not use `.all` in this PR or changing Sponsor to use `.all`